src/static/js/utils/tile-map.js

```diff
@@ -1,19 +1,26 @@
 'use strict';
 
 function getColor( value ) {
+
+  var green50 = '#bae0a2';
```

Comment:
Looks like some of these are duplicates from `TileMap.js`. Could we put all the vars into one config file?